en/components/grids_templates/cell-editing.md

```diff
@@ -165,12 +165,25 @@ You can see and learn more for default cell editing templates in the [general ed
 If you want to provide a custom template which will be applied when a cell is in edit mode, you can make use of the [`igxCellEditor` directive]({environment:angularApiUrl}/classes/igxcelltemplatedirective.html). To do this, you need to pass an `ng-template` with the `igxCellEditor` directive and properly bind your custom control to the [`cell.editValue`]({environment:angularApiUrl}/classes/igxcellcomponent.html#editvalue):
 
 ```html
-<igx-column field="ReorderLevel" header="Reorder Level" [editable]="true">
-    <ng-template igxCellEditor let-cell="cell">
-        <input type="number" [(ngModel)]="cell.editValue" />
+<igx-column field="class" header="Class" [editable]="true" [filterable]="false" width="20%">
+    <ng-template igxCellEditor let-cell="cell" let-value>
+        <igx-select [(ngModel)]="cell.editValue" width="100%" [igxFocus]="true" [type]="'search'">
+            <igx-select-item *ngFor="let class of classes" [value]="class">
+                {{ class }}</igx-select-item>
+        </igx-select>
     </ng-template>
 </igx-column>
 ```
+This code is used in the sample below which implements an [`IgxSelectComponent`](../select.md) component in the cells of the `Race`, `Class` and `Alignment` columns.
```

Comment:
"__an [`IgxSelectComponent`](../select.md) component_" ..I suggest drop the component afterwards as it kind of duplocates when reading ..  _ _an [`IgxSelectComponent`](../select.md) in the cells_